client/src/components/DetailsEvent/index.js

```diff
@@ -1,13 +1,180 @@
-import React from 'react'
-import Header from '../Header/header';
-class DetailsEvent extends React.Component {
+import React, { Component } from "react";
+import NavBar from "../NavBar";
+import SideBar from "../SideBar";
+import axios from "axios";
+
+class DetailsEvent extends Component {
+  state = {
+    id: this.props.history.location.event.id,
```

Comment:
```id``` is not a general variable to put it in the state, you have just to get the passed id from ```params```, so define it where you are going to use

Reply:
done